src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -612,7 +612,7 @@
 
 (rf/reg-event-fx :wallet/clean-up-transaction-flow
  (fn [_]
-   {:fx [[:dispatch [:dismiss-modal :screen/wallet.transaction-confirmation]]
```

Comment:
`[:dispatch [:dismiss-modal :screen/wallet.transaction-confirmation]` was moved to `:wallet/end-transaction-flow`, since it doesn't belong in the state cleanup event.